### PR TITLE
stars added

### PR DESCRIPTION
**Pull request by rostyslav48**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
```
Comment (Anna-Tkachenko) [inline src/index.html:12]: Use semantic tags instead of `div` ones

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,52 @@
...
+      <div class="stars__star stars__star--active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars__star stars__star--active"></div>
+      <div class="stars__star stars__star--active"></div>
```
Comment (Anna-Tkachenko) [inline src/index.html:29]: You should manage active stars only with modifier for `stars` block
You should not add any extra classes for the stars elements

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Add demo link and test report link to pull request description

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+body {
+  margin: 0;
+}
+
+/* #stars region */
```
Comment (dffUqp) [inline src/style.css:5]: I think this comment is redundant

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  background-image: url(images/star-active.svg);
+
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+
+}
+
+/* #endregion */
```
Comment (dffUqp) [inline src/style.css:52]: same about comment

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+
+}
```
Comment (dffUqp) [inline src/style.css:50]: you can combine these selectors
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
}
```

**Review by dffUqp — COMMENTED** [review]
LGTM:fire: but needs a few minor fixes:smiley:

File: src/style.css (hunk 5 above)
Comment (rostyslav48) [reply src/style.css:50]: Thanks for your review. All changes were applied in the 3rd commit.

**Pull request closed without merging**